<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quteman</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="requestType">
        <item>
         <property name="text">
          <string>GET</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>POST</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PUT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PATCH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DELETE</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addressBar">
        <property name="placeholderText">
         <string>Enter request URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submitButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="paramsTab">
        <attribute name="title">
         <string>Params</string>
        </attribute>
        <layout class="QGridLayout" name="paramsLayout">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="paramsInput">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
            </font>
           </property>
           <property name="placeholderText">
            <string>Key: Value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="authTab">
        <attribute name="title">
         <string>Authorization</string>
        </attribute>
        <layout class="QGridLayout" name="authLayoutOuter">
         <item row="0" column="0">
          <layout class="QGridLayout" name="authLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="authUsernameLabel">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="authPasswordLabel">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="authPasswordInput">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="authUsernameInput"/>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="headersTab">
        <attribute name="title">
         <string>Headers</string>
        </attribute>
        <layout class="QGridLayout" name="headersLayout">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="headersInput">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
            </font>
           </property>
           <property name="placeholderText">
            <string>Key: Value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="bodyTab">
        <attribute name="title">
         <string>Body</string>
        </attribute>
        <layout class="QGridLayout" name="bodyLayout">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="bodyInput">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabResult">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="jsonTab">
        <attribute name="title">
         <string>JSON</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTreeView" name="jsonView">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
            </font>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerMinimumSectionSize">
            <number>100</number>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>200</number>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="rawTab">
        <attribute name="title">
         <string>Raw</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="textBrowser">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="headersResultTab">
        <attribute name="title">
         <string>Headers</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="headersBrowser">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addressBar</sender>
   <signal>returnPressed()</signal>
   <receiver>submitButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>751</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
